Missing uavcan_posix for building

Hello guys!!!

I hope this issue provides more information related with the building process.

I feel a little bit confusing when trying to build this library. I am following the instructions provided in the [README](https://github.com/UAVCAN/platform_specific_components/tree/legacy-v0/linux/libuavcan) for **Libuavcan platform driver for Linux**, but I haven't been able to build it.

**Note:** Before starting the building, I make sure I had all the runtime prerequisites, build prerequisites and suggested packages.

As soon as I start the building, I noted that git submodule update --init --recursive command is not performing any action at all (There is not feedback when running it).

Then when running **make -j4** I got the following error:

```bash
/libuavcan_linux/apps/uavcan_dynamic_node_id_server.cpp:24:10: fatal error: uavcan_posix/dynamic_node_id_server/file_storage_backend.hpp: No such file or directory
 #include <uavcan_posix/dynamic_node_id_server/file_storage_backend.hpp>
```
The error is clear; however, I do not understand how to get uavcan_posix (I did not find it anywhere). Is it generated during building o should I manually include it?. I believe that as soon as I get it, the building process will success.

Please see additional information below:

Branch: legacy-v0
Platform-Specific: linux/libuavcan
OS: Ubuntu 18.04

Thank you!!!

--- a/linux/libuavcan/README.md
+++ b/linux/libuavcan/README.md
@@ -16,6 +16,9 @@ Note that the driver will be able to synchronize the system clock with the netwo
 node application is running with root privileges.
 Please read the class documentation at `include/uavcan_linux/clock.hpp` for details.
 
+The [libuavcan_posix](https://github.com/UAVCAN/libuavcan/tree/a38f193511ad24fa2f05f3ccd5988e3d70ad1a59/libuavcan_drivers/posix)
+driver may come useful in most applications as well.
+
 ## Installation
 
 This part describes how to install libuavcan - including the Linux driver - on a Linux machine.

